src/Pos/Launcher/Param.hs

```diff
@@ -30,7 +34,17 @@ data BaseParams = BaseParams
     { bpLoggingParams   :: !LoggingParams  -- ^ Logger parameters
     } deriving (Show)
 
--- | Contains algorithm specific & storage parameters for Node.
+-- | Network parameters.
+data NetworkParams = NetworkParams
+    { npDiscovery :: !(Either (Set NodeId) KademliaParams)
+    -- ^ If this value is 'Left', then given peers will be used in static mode.
+    -- Otherwise kademlia with given params will be used.
+    , npTcpAddr   :: !TCP.TCPAddr
```

Comment:
Address of what? As far as i remember we now have two addresses -- bind address and "public for other people" address. How is it reflected in this case?